QGCSetup.pri

```diff
@@ -58,6 +58,14 @@ MacBuild {
     }
 }
 
+iOSBuild {
+    # AirMap
+    contains (DEFINES, QGC_AIRMAP_ENABLED) {
+        QMAKE_POST_LINK += && rsync -a $$BASEDIR/libs/airmapd/iOS/Qt.5.11.0/* $$DESTDIR/$${TARGET}.app/Frameworks/
+        #QMAKE_POST_LINK += && install_name_tool -change "@rpath/libairmap-qt.0.0.1.dylib" "@executable_path/../Frameworks/libairmap-qt.0.0.1.dylib" $$DESTDIR/$${TARGET}.app/Contents/iOS/$${TARGET}
```

Comment:
I believe that my `rpath` (coming from airmapd) is already correct, so this may be unnecessary here. Needs to be confirmed, though.